.github/actions/prepare/action.yml

```diff
@@ -0,0 +1,21 @@
+name: Prepare test
+description: Prepare an SDK test
+
+inputs:
+  node:
+    description: The Node version to use
+    required: false
+    default: 22
+  cache:
+    description: The cache key identifier to use
+    required: true
+
+runs:
+  using: composite
+
+  steps:
+    - name: Setup Node.js with pnpm caching
+      uses: actions/setup-node@v4
+      with:
+        node-version: ${{ inputs.node }}
+        cache: 'pnpm'
```

Comment:
The original prepare action [also restored the cache](https://github.com/nuxt/nuxt/blob/main/.github/workflows/ci.yml#L354), is this intentional to no longer do that?